src/formatly.ts

```diff
@@ -1,10 +1,15 @@
 import { execa, Result } from "execa";
 
-import { Formatter } from "./formatters.js";
+import { Formatter, FormatterName, formatters } from "./formatters.js";
 import { resolveFormatter } from "./resolveFormatter.js";
 
 export interface FormatlyOptions {
 	cwd?: string;
+
+	/**
+	 * Pass an explicitly formatter to use instead of automatically detecting
+	 */
+	formatter?: FormatterName;
```

Comment:
[Praise] I like the option name `formatter`. Clear & simple, plus it leaves open the possibility that #11 would make it `FormatterName | Formatter` or similar. 👍